test/setup-beam.test.js

```diff
@@ -526,6 +566,34 @@ async function testGetVersionFromSpec() {
   got = setupBeam.getVersionFromSpec(spec, versions)
   assert.deepStrictEqual(got, expected)
   simulateInput('version-type', before)
+
+  spec = 'latest'
+  expected = '24.0.0'
+  got = setupBeam.getVersionFromSpec(spec, versions)
+  assert.deepStrictEqual(got, expected)
+
+  spec = 'latest'
+  got = setupBeam.getVersionFromSpec(spec, [
+    { '22.3.4.2.1.0.1': '22.3.4.2.1.0.1', '22.3.4.2.1.0.0': '22.3.4.2.1.0.0' },
+  ])
+  assert.deepStrictEqual(got, '22.3.4.2.1.0.1')
+
+  got = setupBeam.getVersionFromSpec(spec, matrix.otp['ubuntu-18.04'])
+  assert.deepStrictEqual(got, '26.2.1')
```

Comment:
As per the previous comment, the following structure should be adopted:
```
spec
expected
got
assert.
<newline>
```

(and you should repeat `spec` even if it's always `latest` so that the blocks make sense by themselves and we don't have to scroll to understand what we're testing)